src/test/java/teammates/test/cases/webapi/GetAuthInfoActionTest.java

```diff
@@ -125,9 +130,66 @@ protected void testExecute() {
         assertThrows(UnauthorizedAccessException.class, () -> getAction(new String[] {
                 Const.ParamsNames.USER_ID, "idOfAnotherInstructor",
         }));
+    }
+
+    @Test
+    public void testExecute_addCsrfTokenCookies_shouldAddToResponseAccordingToExistingCsrfToken() {
+
+        ______TS("Test will add CSRF token cookies: No logged in user");
```

Comment:
What does "Test will add CSRF token cookies" really means? Just `No logged in user` is enough to describe the scenario?